.yarn/patches/@lavamoat-lavapack-npm-3.3.0-5a61b5374d.patch

```diff
@@ -0,0 +1,55 @@
+diff --git a/src/runtime.js b/src/runtime.js
+index 97c9ddb9b76a2fe31cf9b202cb491f15232232ff..02d657e84d6d4572d9b6eb5e73caa01380c89b0c 100644
+--- a/src/runtime.js
++++ b/src/runtime.js
+@@ -10831,6 +10831,10 @@ function makeGetEndowmentsForConfig ({ createFunctionWrapper }) {
+ function getPropertyDescriptorDeep (target, key) {
+   let receiver = target
+   while (true) {
++    // abort if this is the end of the prototype chain.
++    if (!receiver) {
++      return { prop: null, receiver: null }
++    }
+     // support lookup on objects and primitives
+     const typeofReceiver = typeof receiver
+     if (typeofReceiver === 'object' || typeofReceiver === 'function') {
+@@ -10845,10 +10849,6 @@ function getPropertyDescriptorDeep (target, key) {
+       // eslint-disable-next-line no-proto
+       receiver = receiver.__proto__
+     }
+-    // abort if this is the end of the prototype chain.
+-    if (!receiver) {
+-      return { prop: null, receiver: null }
+-    }
+   }
+ }
+ 
+@@ -11384,7 +11384,9 @@ module.exports = {
```

Comment:
change below is discussed here https://github.com/LavaMoat/LavaMoat/pull/464

Reply:
and in that context, how was [this](https://github.com/LavaMoat/LavaMoat/pull/464/files#diff-cbec1016c903fe52b8d76860ebf3400e172b83a3374b8796bab149b3721d8064R465) not a problem to you @FrederikBolding?

Reply:
This is the weirdest thing, I can't reproduce the error that got me working on this solution...
Let's narrow down back to try-catch solution and get back to the broader solution if the problem pops up again..

ctx: https://github.com/LavaMoat/LavaMoat/pull/464#issuecomment-1462302562
new pr: https://github.com/LavaMoat/LavaMoat/pull/468